spec/unit/data_collector_spec.rb

```diff
@@ -23,6 +23,14 @@
 require "chef/resource_builder"
 
 describe Chef::DataCollector do
+
+  # TODO:
```

Comment:
Are these still open? (I'm not sure what exactly you tracked here, but it seems like these have been taken care of by this change, doesn't it?)